src/JNI/com/amazonaws/kinesis/video/producer/jni/Parameters.cpp

```diff
@@ -201,6 +201,22 @@ BOOL setClientInfo(JNIEnv *env, jobject clientInfo, PClientInfo pClientInfo) {
         CHK_JVM_EXCEPTION(env);
     }
 
+    methodId = env->GetMethodID(cls, "getServiceConnectionTimeout", "()J");
+    if (methodId == NULL) {
+        DLOGW("Couldn't find method id getServiceConnectionTimeout");
+    } else {
+        pClientInfo->serviceCallConnectionTimeout = env->CallLongMethod(clientInfo, methodId);
+        CHK_JVM_EXCEPTION(env);
+    }
+
```

Comment:
We are yet to implement this in the Java SDK, right?

Reply:
We already have a PR in Java for this: https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/pull/182